lib/super_digit.rb

```diff
@@ -1,10 +1,22 @@
 # Superdigit
 
-# Time Complexity - ?
-# Space Complexity - ?
+# Time Complexity - O(n)
+# Space Complexity - O(n)
 def super_digit(n)
```

Comment:
Because summing is O(n) where n is the number of digits and you're doing that inside the recursion, this solution is O(n^2).